ethicml/implementations/adv_debiasing_method.py

```diff
@@ -33,7 +33,7 @@ def fit(train: DataTuple, args: AdvDebArgs, seed: int = 888) -> AdvDebiasingClas
         cost_pred=torch.nn.CrossEntropyLoss(),
         in_shape=len(train.x.columns),
         batch_size=args["batch_size"],
-        model_type=args["model_type"],
+        model_type=ModelType(args["model_type"]),
```

Comment:
i always get thrown about when `Enum(value)` is needed and `Enum[value]` - though the tests would tell us pretty quickly if there were a problem 😅

Reply:
the tests do indeed fail, so let me check

Reply:
So, it's `Enum(value)` and `Enum[name]`, but I'm using `StrEnum` here, so the value is just the lower case version of the name.

So:
```python
class E(StrEnum):
    A = auto()
    B = auto()

# this works:
E("a")
# and this works:
E["A"]
```